command/oauth/cmd.go

```diff
@@ -336,7 +352,20 @@ func oauthCmd(c *cli.Context) error {
 		prompt = c.String("prompt")
 	}
 
-	o, err := newOauth(opts.Provider, clientID, clientSecret, authzEp, tokenEp, scope, prompt, opts)
+	authParams := map[string]string{}
+	for _, keyval := range c.StringSlice("auth-param") {
+		parts := strings.Split(keyval, "=")
+		if len(parts) != 2 {
+			return errs.InvalidFlagValue(c, "auth-param", keyval, "")
+		}
```

Comment:
It is not common, but we can be a little more permissive and use SplitN, so weird parameters like `--auth-param foo=bar=zar` will work, key will be `foo` and the value `bar=zar`